src/pedalintandem.py

```diff
@@ -0,0 +1,114 @@
+import json
+from datetime import datetime
+from curl_cffi import requests
+from common.tz import IST
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://www.pedalintandem.com"
+
+def _date(date_str):
+    return datetime.fromisoformat(date_str).astimezone(IST).isoformat()
+
+
+def fetch_events_links(session):
+	res = session.get(f"{BASE_URL}/experiences", impersonate="chrome")
+	soup = BeautifulSoup(res.text, 'html.parser')
+	event_divs = soup.find_all('div', class_ = 'single-experience')
+
+	# Fetch events from other pages.
```

Comment:
Took me a while to figure out how/why this works. Would be nice to document with a comment:

PIT supports their infinite scroll without Javascript by pagination. We paginate through all these pages and collect the event URLs

Reply:
Ok. Will add more comments.

Yes. Javascript was turned off that is why infinite scroll was not working. I will change this logic.

Reply:
I think we can make the assumption that important events (happening in bangalore, featured, or upcoming) will always show up on the main page. And drop the entire pagination code since that just adds complexity. So the code becomes:

1. Fetch home page
2. find all experience links
3. Iterate/Transform
4. Generate